py21cmsense/sensitivity.py

```diff
@@ -93,11 +101,10 @@ def _kconverter(val):
     if not hasattr(val, "unit"):
         # Assume it has the 1/Mpc units (default from 21cmFAST)
         return val * un.littleh / un.Mpc / Planck15.h
-    else:
-        try:
-            return val.to("littleh/Mpc")
-        except un.UnitConversionError:
-            return ((val / Planck15.h) * un.littleh).to("littleh/Mpc")
+    try:
+        return val.to("littleh/Mpc")
+    except un.UnitConversionError:
+        return ((val / Planck15.h) * un.littleh).to("littleh/Mpc")
```

Comment:
I thought we changed this to use the `with_H0` equivalency somewhere. am i just making that up?

Reply:
I found it, it's in #47  which is unmerged at the time of writing.